src/App.js

```diff
@@ -1,25 +1,70 @@
-import logo from './logo.svg';
 import './App.css';
+import { useState } from 'react';
+import DisplayActivity from './Components/DisplayActivity';
+import Form from './Components/Form';
+import Error from './Components/Error';
 
 function App() {
+
+  const [activity, setActivity] = useState("");
+  const [activityLink, setActivityLink] = useState("");
+  const [formError, setFormError] = useState(false);
+
+  const getActivity = (numberOfKids, typeOfActivity) => {
+
+  console.log(numberOfKids)
+  console.log(typeOfActivity)
+    const url = new URL("https://www.boredapi.com/api/activity");
+    
+    url.search = new URLSearchParams({
+      "participants": numberOfKids,
+      "type": typeOfActivity,
+      "minprice" : 0, 
+      "maxprice" : 0.5,
+      "minaccessibility": 0,
+      "maxaccessibility": 0.4
+    })
+
+    fetch(url)
+      .then(data=> 
+        {console.log(data)
+         return data.json()})
+      .then((response) => {
+        console.log(response);
+        const activityToDisplay = response.activity;
+        const activityLink = response.link;
```

Comment:
I love the inclusion of a link to find out more information! Shame that hardly any activities have anything in that field.